io.github.stella_emu.Stella.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+      <image>https://stella-emu.github.io/game-images/chopper_command.png</image>
+      <caption>Stella running 'Chopper Command' (Normal rendering)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://stella-emu.github.io/game-images/keystone.png</image>
+      <caption>Stella running 'Keystone Kapers' (Normal rendering)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://stella-emu.github.io/game-images/berzerk.png</image>
+      <caption>Stella running 'Berzerk' (Blargg TV Effects)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://stella-emu.github.io/game-images/hero.png</image>
+      <caption>Stella running 'H.E.R.O.' (Blargg TV Effects)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://stella-emu.github.io/game-images/debugger.png</image>
+      <caption>Integrated Debugger</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://stella-emu.github.io</url>
+  <update_contact>scummvm-devel@lists.scummvm.org</update_contact>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
```

Comment:
`none` fields can be skipped.

Reply:
Took this one from ScummVM...

Since all entries are "none" should I remove "<content_reting...>" completely or keep it with no entries?

Reply:
It should stay, just without entries. Probably can be reduced to single line with `</>`.